### PR TITLE
Improve grid subcommands usability / unify logic

### PR DESCRIPTION
**Pull request by jakolehm**

### CODE REVIEW

**Comment by nevalla** [comment]
@jakolehm Still work in progress?

**Comment by jakolehm** [comment]
Yes, functionality is there but specs are not updated.

**Comment by nevalla** [comment]
@jakolehm this needs rebase

**Comment by SpComb** [comment]
:+1: on the idea, but we should probably preserve the existing argument order.

Needs updates for conflicts.

**Comment by jnummelin** [comment]
Still valid?

**Comment by kke** [comment]
Partially.

File: cli/lib/kontena/cli/grids/cloud_config_command.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ class CloudConfigCommand < Kontena::Command
     include Kontena::Cli::Common
     include Common
 
-    parameter "NAME", "Grid name"
+    parameter "[NAME]", "Grid name (default: current grid)"
```
Comment (SpComb) [inline cli/lib/kontena/cli/grids/cloud_config_command.rb]: Making this optional still looks legit.

**Comment by SpComb** [comment]
Obsolete, and #2294 fixed the worst `kontena grid trusted-subet` bits. Some stuff that could still be salvaged here, but that would just be a new PR at this point.

**Pull request closed without merging**